app/models/app_resource_monitor/postgres_inspector.rb

```diff
@@ -116,23 +116,34 @@ def query_variables
   INDEX_STATS_SQL = <<~SQL.freeze
     SELECT
       %{db_name} AS database,
-      t.tablename,
+      c.relname AS tablename,
```

Comment:
https://github.com/greenriver/hmis-warehouse/blob/stable/app/models/dba/database_bloat.rb?plain=1#L61
and related code in that file might be unified with this.